src/content/docs/codestream/start-here/sign-up-codestream.mdx

```diff
@@ -3,48 +3,63 @@ title: Sign up for CodeStream
 description: "An overview of CodeStream's accounts and organizations and how to set those up for your team."
 ---
 
-To get the most out of New Relic CodeStream's collaboration tools, create an organization and then invite your team members to it. 
+To get the most out of New Relic CodeStream's collaboration tools, create an
+organization and then invite your team members to it. 
 
-If you haven't already, [sign up for a free New Relic account](https://newrelic.com/signup) to jump from your application's stack trace errors in New Relic One directly to the line of code responsible in your IDE.
+If you haven't already, [sign up for a free New Relic
+account](https://newrelic.com/signup) to jump from your application's stack
+trace errors in New Relic One directly to the line of code responsible in your
+IDE.
 
 ## Create an account [#account]
 
-If you already have the CodeStream extension [installed in your IDE](/docs/codestream/start-here/install-codestream), you can start the sign up process from the CodeStream pane.
+If you already have the CodeStream extension [installed in your
+IDE](/docs/codestream/start-here/install-codestream), you can start the sign up
+process from the CodeStream pane.
 
 ![Create an account](./images/CreateAnAccount6.png)
 
-There are two ways to sign up for CodeStream. You can sign up with a set of CodeStream credentials (such as, email address and password). Alternatively, you can sign up using your GitHub, GitLab, or Bitbucket (cloud versions only) account. 
+You can sign up using your email address and a password, or you can sign up using
+your GitHub, GitLab, or Bitbucket account (cloud versions only, although
+CodeStream does have integrations with the on-prem versions once you've
+signed up). If you're a New Relic customer, you can also sign up with a user api
```

Comment:
```suggestion
your GitHub, GitLab, or Bitbucket account. If you're a New Relic customer, you can also sign up with a user api
```

I think we should drop this parenthetical. If this is important enough to warrant clarifying, let's create a brief troubleshooting doc to address this and then link to it from here.